Sources/Sentry/SentryReachability.m

```diff
@@ -109,9 +116,8 @@ + (void)initialize
 
 - (void)dealloc
 {
-    for (id<SentryReachabilityObserver> observer in sentry_reachability_change_blocks.allKeys) {
-        [self removeObserver:observer];
-    }
```

Comment:
This make reachability flaky during tests.

Reply:
Shouldn't this change be in an extra PR?